includes/class-wc-tracker.php

```diff
@@ -34,10 +38,10 @@ public static function init() {
 	/**
 	 * Decide whether to send tracking data or not.
 	 *
-	 * @param boolean $override
+	 * @param boolean $override Should override?.
 	 */
 	public static function send_tracking_data( $override = false ) {
-		// Don't trigger this on AJAX Requests
+		// Don't trigger this on AJAX Requests.
 		if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
```

Comment:
Why is this needed? CRON is not an AJAX request and thats the only time this is called right?

Reply:
It was added because it was firing multiple times without this, think it has to do with the initial send which is not done via cron.